src/cargo/util/toml/mod.rs

```diff
@@ -1737,12 +1738,24 @@ impl TomlManifest {
             debug!("manifest has no build targets");
         }
 
-        if let Err(e) = unique_build_targets(&targets, package_root) {
-            warnings.push(format!(
-                "file found to be present in multiple \
-                 build targets: {}",
-                e
-            ));
+        if let Err(conflict_targets) = unique_build_targets(&targets, package_root) {
+            conflict_targets
+                .iter()
+                .for_each(|(target_path, conflicts)| {
+                    warnings.push(format!(
+                        "file `{}` found to be present in multiple \
```

Comment:
Just noticed that the term `build target` is not accurate and could be mistaken for config [`build.target`](https://doc.rust-lang.org/nightly/cargo/reference/config.html). No need to fix it in this pull request (it is all over this file 😆)